tensorflow/stream_executor/cuda/cuda_dnn.cc

```diff
@@ -3394,6 +3391,21 @@ std::tuple<int, int> GetTensorVectorSizeAndDim(
   return std::make_tuple(vector_size, vector_dim);
 }
 
+cudnn_frontend::Tensor CreateCudnnTensor(
+    const int64_t* dims, const int64_t* strides, int64_t ndims, int64_t uid,
+    cudnnDataType_t dtype, int64_t vec_count, int64_t vec_dim,
```

Comment:
Every call to this passes a type that came from ToCudnnDataType(something), and most of those are unused except in calls to CreateCudnnTensor -- so I think it'd be nicer to have the conversion internal to this function, and eliminate several of the variables like cudnn_input_type.